### PR TITLE
Fixes annoying issues + Remaps a jungle ruin

### PR DESCRIPTION
**Pull request by Zevotech**
## About The Pull Request
Makes goliaths no longer shove wrenched windows out of place. Goliaths also now drop 2 hide plates on butcher instead of four, meaning to fully armor an explorer suit or hardsuit you will only need to kill three goliaths for plates.
Makes Ice Whelps no longer shove wrenched windows and break down every wall in your ship.
Remaps jungle_botany.dmm and removes the ERT hardsuits which were accidentally placed in its suit storage units.

## Why It's Good For The Game
Goliaths and Ice Whelps shoving wrenched windows was a known issue and I decided to get off of my ass and fix it. This is bundled with a remap of a ruin because its 11:50 PM and im really fucking tired.
image

## Changelog
:cl:
balance: Remapped Jungle_Botany.dmm, removed its ERT hardsuits
balance: Made Ice Whelps no longer tear down walls and murder entire ships (Now also applies to polar bears)
balance: Lowered the amount of hide goliaths drop from 4 to 2, as requested by a lead
fix: fixed goliaths, ice whelps, polar bears and ice demons shoving wrenched windows
/:cl:

### CODE REVIEW

**Comment by Zevotech** [comment]
oh thank shitfuck the checks passed time to sleep

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm [hunk 1]
```diff
@@ -32,15 +32,17 @@
...
-	pull_force = MOVE_FORCE_VERY_STRONG
+	pull_force = MOVE_FORCE_NORMAL
 	gender = MALE //lavaland elite goliath says that it s female and i s stronger because of sexual dimorphism, so normal goliaths should be male
 	var/can_charge = TRUE
 	var/pre_attack = 0
 	var/pre_attack_icon = "Goliath_preattack"
 	var/tentacle_type = /obj/effect/temp_visual/goliath_tentacle
-	loot = list(/obj/item/stack/sheet/animalhide/goliath_hide)
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab/goliath = 2, /obj/item/stack/sheet/bone = 2, /obj/item/stack/sheet/sinew = 2, /obj/item/stack/ore/silver = 10)
+	guaranteed_butcher_results = list(/obj/item/stack/sheet/animalhide/goliath_hide = 2)
```
Comment (ZephyrTFA) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm:42]: undocumented changes

Comment (Zevotech) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm:42]: zeta asked me to change that and i mused to myself "im totally going to forget to put this in the changelog" and here we are. here we fucking are.

Comment (Zevotech) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm:42]: fixed

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm [hunk 2]
```diff
@@ -62,15 +64,15 @@
...
 		move_resist = MOVE_FORCE_VERY_STRONG
-		pull_force = MOVE_FORCE_VERY_STRONG
+		pull_force = MOVE_FORCE_NORMAL
 		. = 1
 
 /mob/living/simple_animal/hostile/asteroid/goliath/death(gibbed)
 	move_force = MOVE_FORCE_DEFAULT
 	move_resist = MOVE_RESIST_DEFAULT
-	pull_force = PULL_FORCE_DEFAULT
+	pull_force = MOVE_FORCE_DEFAULT
```
Comment (Fikou) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm]: most of this code is useless now

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm [hunk 3]
```diff
@@ -62,15 +64,15 @@
 
 /mob/living/simple_animal/hostile/asteroid/goliath/revive(full_heal = FALSE, admin_revive = FALSE)//who the fuck anchors mobs
 	if(..())
-		move_force = MOVE_FORCE_VERY_STRONG
+		move_force = MOVE_FORCE_NORMAL
 		move_resist = MOVE_FORCE_VERY_STRONG
-		pull_force = MOVE_FORCE_VERY_STRONG
+		pull_force = MOVE_FORCE_NORMAL
```
Comment (Fikou) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm]: and this

**Review by Fikou — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm (hunk 2 above)
Comment (Zevotech) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm:75]: ok?

**Comment by Zevotech** [comment]
done, golaiths cant push anchored windows anymore. we are finally free.

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm [hunk 4]
```diff
@@ -32,15 +32,17 @@
 	throw_message = "does nothing to the rocky hide of the"
 	vision_range = 5
 	aggro_vision_range = 9
-	move_force = MOVE_FORCE_VERY_STRONG
+	move_force = MOVE_FORCE_NORMAL
 	move_resist = MOVE_FORCE_VERY_STRONG
-	pull_force = MOVE_FORCE_VERY_STRONG
+	pull_force = MOVE_FORCE_NORMAL
```
Comment (Fikou) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm]: move_force and pull_force are inherited from parent, you dont want to set them

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm [hunk 5]
```diff
@@ -62,15 +64,9 @@
...
-		move_force = MOVE_FORCE_VERY_STRONG
-		move_resist = MOVE_FORCE_VERY_STRONG
-		pull_force = MOVE_FORCE_VERY_STRONG
 		. = 1
 
 /mob/living/simple_animal/hostile/asteroid/goliath/death(gibbed)
-	move_force = MOVE_FORCE_DEFAULT
-	move_resist = MOVE_RESIST_DEFAULT
-	pull_force = PULL_FORCE_DEFAULT
 	..()
```
Comment (Fikou) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm]: dont make overrides of procs that only call parent. also move_resist still needs to be set/unset from normal

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm [hunk 6]
```diff
@@ -62,15 +64,9 @@
 
 /mob/living/simple_animal/hostile/asteroid/goliath/revive(full_heal = FALSE, admin_revive = FALSE)//who the fuck anchors mobs
 	if(..())
-		move_force = MOVE_FORCE_VERY_STRONG
-		move_resist = MOVE_FORCE_VERY_STRONG
-		pull_force = MOVE_FORCE_VERY_STRONG
 		. = 1
```
Comment (Fikou) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm]: this'll be return TRUE

Comment (Fikou) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm]: or you can replace if(..()) with . = ..() and if(.)

**Comment by Zevotech** [comment]
i dont know anymore i just want this to get merged so i dont have to hear about ice whelps ripping through entire ships

**Review by Fikou — APPROVED** [review]

**Pull request merged**